R/plot.R

```diff
@@ -28,6 +28,7 @@ autoplot.cre <- function(object, ...){
   cate_results <- object[["CATE"]]
   cate_method <- object[["cate_method"]]
   M <- object[["M"]]
+  if (M["Causal (significant)"]==0){stop("Visualization not available (0 causal decision rules discovered).")}
```

Comment:
Move this check under the `plot.cre` function and convert it into a message rather than an error.

Reply:
In order to move the message to `plot.cre`, I unpack `...` there as well.

Reply:
fixed